cmd/wego/add/cmd.go

```diff
@@ -58,12 +64,29 @@ func runCmd(cmd *cobra.Command, args []string) {
 		shims.Exit(1)
 	}
 
+	if params.Url == "" {
+		if len(args) == 0 {
+			fmt.Fprint(shims.Stderr(), "no app --url or app location specified")
+			shims.Exit(1)
+		} else {
+			params.Dir = args[0]
+		}
+	}
+
 	gitClient := git.New(authMethod)
+	fluxClient := flux.New()
+	kubeClient := kube.New()
+	gitProviders := gitproviders.New()
 
-	deps := &cmdimpl.AddDependencies{
-		GitClient: gitClient,
+	deps := &app.Dependencies{
```

Comment:
:+1: I like this. It does make me wonder if the `cmdimpl` package is necessary, since it just wraps the library.

Reply:
no, after the migration we can delete that.